README.md

```diff
@@ -1 +1,70 @@
-# Math-Magicians
\ No newline at end of file
+# Getting Started with Create React App
+
```

Comment:
- [ ] It would be better to create a professional README file, a professional README file should give details of the project instead of the one we have currently. Kindly check this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) for more information